portal-impl/src/com/liferay/portal/model/impl/CountryImpl.java

```diff
@@ -48,7 +48,17 @@ public String getNameCurrentLanguageId() {
 	@JSON
 	@Override
 	public String getNameCurrentValue() {
-		return getName(getLocale(_nameCurrentLanguageId));
+
```

Comment:
Please run formatSource

Reply:
@sontruongces I've already run format source, what is the problem here?

Reply:
ci:test:sf failed in the PR: https://github.com/liferay-usm/liferay-portal/pull/733. Shouldn't have an empty line here (line 51).

Reply:
ok, thanks for letting me know. Updated.